packages/cli/src/generators/fmdapi.ts

```
@@ -0,0 +1,529 @@
+
+  let configArray: ImportedDataSourceConfig[];
+  if (Array.isArray(fileContent.config)) {
+    configArray = fileContent.config;
+  } else {
+    configArray = [fileContent.config];
+    fileContent.config = configArray;
+  }
+
+  const targetDataSource = configArray.find((ds) => ds.path === targetDsPath);
+
+  if (targetDataSource && targetDataSource.layouts) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)